cromshell

```diff
@@ -891,11 +888,11 @@ function pretty-execution-status()
   local -r workflowId=${1} metadataFile=${2} expandSubWorkflows=${3:-false}
   local -r workflowStatus=$(jq -r .status ${metadataFile})
   echo -e "${COLOR_UNDERLINED}${workflowId}\t${workflowStatus}${COLOR_NORM}"
-  printWorklowStatus ${metadataFile} "" ${expandSubWorkflows}
+  printWorkflowStatus ${metadataFile} "" ${expandSubWorkflows}
 }
 
 # Print a workflow (or sub-workflow) pretty-execution-status summary
-printWorklowStatus()
+printWorkflowStatus()
```

Comment:
For consistency this should be `function printWorkflowStatus()`.

Reply:
I was wondering about that, I wasn't sure what specifying function did.